src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -115,34 +104,34 @@ void givenValidAdminUserRegisterRequest_thenAdminUserRegistered() {
     }
 
     @Test
-    void givenInvalidApplicationIdFromAdminUserRegisterRequest_whenApplicationEntityNotFound_thenThrowAysAdminUserRegisterApplicationCodeNotValidException() {
+    void givenInvalidApplicationId_whenApplicationEntityNotFound_thenThrowAysAdminUserRegisterApplicationNotExistByIdAndStatusException() {
 
         // Given
-        AdminUserRegisterRequest mockAdminUserRegisterRequest = new AdminUserRegisterRequestBuilder()
-                .withApplicationId("Invalid").build();
+        String applicationId = "Invalid";
+        AdminUserRegisterApplicationCompleteRequest mockAdminUserRegisterApplicationCompleteRequest = new AdminUserRegisterRequestBuilder()
+                .withValidFields().build();
 
         // When
         Mockito.when(adminUserRegisterApplicationRepository.findById(Mockito.anyString()))
                 .thenReturn(Optional.empty());
 
         // Then
         Assertions.assertThrows(
-                AysAdminUserRegisterApplicationCodeNotValidException.class,
-                () -> adminUserRegisterService.register(mockAdminUserRegisterRequest)
+                AysAdminUserRegisterApplicationNotExistByIdAndStatusException.class,
+                () -> adminUserRegisterService.completeRegistration(applicationId, mockAdminUserRegisterApplicationCompleteRequest)
         );
 
         Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Ekledim.

Reply:
✔ Bu kısmı **Resolve Conversation** edebiliriz.